rasa_nlu/featurizers/regex_featurizer.py

```diff
@@ -43,8 +43,8 @@ def __init__(self, component_config=None, known_patterns=None):
     def train(self, training_data, config, **kwargs):
         # type: (TrainingData, RasaNLUModelConfig, **Any) -> None
 
-        for example in training_data.regex_features:
-            self.known_patterns.append(example)
+        self.known_patterns = [pattern
```

Comment:
`training_data.regex_features[:]`?

Reply:
or why not `training_data.regex_features`?

Reply:
we never change it

Reply:
sure, just want to get rid of that useless loop :D